GitHubContentUtility/Operations/PullRequestCreator.cs

```diff
@@ -0,0 +1,77 @@
+﻿// ------------------------------------------------------------------------------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------------------------------------------------------------------------------
+
+using Octokit;
+using System;
+using System.Threading.Tasks;
+using GitHubContentUtility.Common;
+using GitHubContentUtility.Services;
+using System.Linq;
+
+namespace GitHubContentUtility.Operations
+{
+    /// <summary>
+    /// Provides method for writing to a blob in a GitHub repository.
+    /// </summary>
+    public class PullRequestCreator
```

Comment:
any reason this isn't static?  no instance variables

Reply:
None. Static classes will work perfectly.